components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -702,6 +702,7 @@ void KeyringController::Reset() {
   default_keyring_.reset();
 
   prefs_->ClearPref(kBraveWalletKeyrings);
+  prefs_->ClearPref(kBraveWalletCustomNetworks);
```

Comment:
I realize we want to clear this at the same time as clearing the keyring. I wonder if we should be doing this somewhere else though because keyring_controller.cc doesn't own the pref `kBraveWalletCustomNetworks` @darkdh ?

Reply:
I was suggesting calling RPCController static function to clear the pref it owns. And somehow it became this
https://github.com/brave/brave-core/pull/9798#issuecomment-911235701

Reply:
static function sgtm